scripts/pre_commit_linter.py

```diff
@@ -442,17 +492,27 @@ def _pre_commit_linter(all_files):
         parent_dir, 'oppia_tools', 'node-6.9.1', 'bin', 'node')
     eslint_path = os.path.join(
         parent_dir, 'node_modules', 'eslint', 'bin', 'eslint.js')
-
-    if not os.path.exists(eslint_path):
+    stylelint_path = os.path.join(
+        parent_dir, 'node_modules', 'stylelint', 'bin', 'stylelint.js')
+    if not (os.path.exists(eslint_path) and os.path.exists(stylelint_path)):
         print ''
         print 'ERROR    Please run start.sh first to install node-eslint '
-        print '         and its dependencies.'
+        print '         or node-stylelint and its dependencies.'
         sys.exit(1)
 
     js_files_to_lint = [
         filename for filename in all_files if filename.endswith('.js')]
     py_files_to_lint = [
         filename for filename in all_files if filename.endswith('.py')]
+    css_files_to_lint = [
+        filename for filename in all_files if filename.endswith('oppia.css')]
+
+    css_result = multiprocessing.Queue()
+    css_stdout = multiprocessing.Queue()
+    css_linting_process = multiprocessing.Process(
+        target=_lint_css_files, args=(stylelint_path, css_files_to_lint,
+                                      css_stdout, css_result))
+    css_linting_process.daemon = True
```

Comment:
Just to check, why is this needed here and not in the js/py linting? Should we add it to the others too (or remove this one)?

Reply:
OK, I would suggest just making sure that all three checks are as consistent as possible.

Reply:
Done, thanks!